stoplight.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function(){
+  // $(".bulb").click(function(){
+  //   $(this).toggleClass("active");
+  // });
+  $(".bulb").click(function(){
+    var thisBulb = $(this).attr('id');
+    var otherBulbs = [];
+    switch (thisBulb) {
+      case "green":
+        otherBulbs.push("yellow", "red");
+        break;
+      case "yellow":
+        otherBulbs.push("green", "red");
+        break;
+      case "red":
+        otherBulbs.push("green", "yellow");
+        break;
+    }
+
+    $(this).addClass("active");
+    otherBulbs.forEach(function(bulb){
```

Comment:
You could also remove the `active` class from all bulbs prior to lighting up the correct one.